projects/guava/UrlEscapersFuzzer.java

```diff
@@ -0,0 +1,88 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueHigh;
+import com.google.common.base.Charsets;
+import com.google.common.escape.Escaper;
+import com.google.common.net.UrlEscapers;
+import java.lang.IllegalArgumentException;
+import java.math.BigInteger;
+import java.net.URLDecoder;
+
+public class UrlEscapersFuzzer {
+
+	/*
+	 * these constants are private members copy-pasted from
+	 * com.google.common.net.UrlEscapers
+	 */
+	static final String URL_FORM_PARAMETER_OTHER_SAFE_CHARS = "-_.*";
+	static final String URL_PATH_OTHER_SAFE_CHARS_LACKING_PLUS =
+      "-._~" // Unreserved characters.
+          + "!$'()*,;&=" // The subdelim characters (excluding '+').
+          + "@:"; // The gendelim characters permitted in paths.
+
+    private static String toHex(String arg) {
+		return String.format("%x", new BigInteger(1, arg.getBytes(Charsets.UTF_8)));
+	}
+
+	private static void rejectSample(String original, String result, String encoded, String percentEncoded) {
+		System.out.println("sample: " + toHex(original));
+		System.out.println("encoded: " + encoded);
+		System.out.println("percentEncoded: " + percentEncoded);
+		System.out.println("result: " + toHex(result));
+		throw new RuntimeException();
+	}
+
+	private static boolean contains(String string, char character) {
+		for (int i=0; i<string.length(); ++i) {
```

Comment:
We don't enforce any coding style we try to be similar to the Google one (https://google.github.io/styleguide/javaguide.html) if possible. In this case if you could just add spaces between operators. Thanks :)